examples/.gitignore

```diff
@@ -0,0 +1,16 @@
+obj/
+bin
+gen
+.settings
+**/.DS_Store
```

Comment:
IDE-specific and OS-specific files should not go into repo `.gitignore` files. People should setup global ignore files for that. (I like to put a header into my repo `.gitignore` files pointing the documentation; e.g. https://github.com/tbroyer/gwt-devserver/blob/master/.gitignore)

For Gradle projects, the `.gitignore` file should generally only include `/.gradle/` and `build/`. For Android projects I suppose `local.properties` and possibly a couple other files are welcome additions.